OK, I have found it. If the file happens not to parse correctly, verify_gofmt spits out a a not very useful message:
`
$ hack/verify-gofmt.sh
!!! Error in hack/verify-gofmt.sh:52
  Error in hack/verify-gofmt.sh:52. 'diff=$(find_files | xargs ${gofmt} -d -s 2>&1)' exited with status 123
Call stack:
  1: hack/verify-gofmt.sh:52 main(...)
Exiting with status 1
`

After my fix you get the following:
`
$ hack/verify-gofmt.sh
./test/e2e/stackdriver_monitoring.go:95:100: missing ',' in argument list
`

I believe the fix makes verify_gofmt more useful, but it is up to you to agree/disagree with me. Let me know what you think.

--- a/hack/verify-gofmt.sh
+++ b/hack/verify-gofmt.sh
@@ -48,7 +48,11 @@ find_files() {
     \) -name '*.go'
 }
 
-diff=$(find_files | xargs ${gofmt} -d -s 2>&1)
+# gofmt exits with non-zero exit code if it finds a problem unrelated to
+# formatting (e.g., a file does not parse correctly). Without "|| true" this
+# would have led to no useful error message from gofmt, because the script would
+# have failed before getting to the "echo" in the block below.
+diff=$(find_files | xargs ${gofmt} -d -s 2>&1) || true
 if [[ -n "${diff}" ]]; then
   echo "${diff}"
   exit 1

